### ISSUE TITLE
Default parameters to run the program，but the output picture is chaotic!!!

### ISSUE THREAD

**Issue by duanliangtao** [milesial/Pytorch-UNet#80]
Thanks for your code,but when i run the demo,the results not as expected！I use the default parameters to run the program，but the output picture is chaotic！！！
![image](https://user-images.githubusercontent.com/48194157/68278491-65c4e980-00ac-11ea-98da-364f435cac0a.png)

**Comment by milesial** [comment]
Hi, for how many epochs did you train the network ? What are you losses ?

**Comment by milesial** [comment]
Hi, can you try pulling the last commit and run the predict script again ?

**Comment by whattheweather** [comment]
> Hi, for how many epochs did you train the network ? What are you losses ?

My output is also bad. I use ~3500 imgs training from scratch with lr=0.005 and epoch=20. The loss comes to about 0.2 at the begining of training, and varies from 0.2 to 0.6. The validation dice coeff raises from 0.2 to 0.4. I have used default settings, the output also seems to be randomly generated.

**Comment by milesial** [comment]
I updated the predict script in the last commit, what you are seeing are the raw logits and not the probabilities

**Comment by whattheweather** [comment]
> I updated the predict script in the last commit, what you are seeing are the raw logits and not the probabilities

Yeah, solved! Thanks!
